<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineWidget</class>
 <widget class="QWidget" name="LineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxSensitivity">
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>12</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>灵敏度</string>
   </property>
   <widget class="QLineEdit" name="txtSensitivity">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubSensitivity">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>50</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSensitivity">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddSensitivity">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>50</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSensitivity_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>122</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>搜索范围</string>
   </property>
   <widget class="QLineEdit" name="txtRange">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRangeDown">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>50</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderRange">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>39</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRangeUp">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>50</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBoxDirect_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>检测方向</string>
    </property>
    <widget class="QRadioButton" name="radioButtonW2B">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>33</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>亮到暗</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonB2W">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>73</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>暗到亮</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAngelLimit">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>130</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>角度门限</string>
    </property>
    <widget class="QPushButton" name="btnSubAngleLimit">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>50</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAngleLimit">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderAngleLimit">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>50</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddAngleLimit">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>50</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDirect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>边沿方向</string>
    </property>
    <widget class="QRadioButton" name="radioButtonPositiveDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>正方向</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonNegativeDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>73</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>负方向</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>角度</string>
    </property>
    <widget class="QLabel" name="labelTheta">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxLineNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>直线数量</string>
   </property>
   <widget class="QPushButton" name="btnLineSub">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLineAdd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextLine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPreLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>51</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCurrentLine">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>51</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtLineNum">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>51</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCoordinate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>坐标微调</string>
   </property>
   <widget class="QPushButton" name="btnMoveRight">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveDown">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveLeft">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveUp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Point1">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>48</y>
      <width>52</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>点a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtPoint1Coor">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>48</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(100,100)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Point2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>111</y>
      <width>52</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>点b:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtPoint2Coor">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>111</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(100,100)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
